Development/src/pages/devices/DevicesList.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import {
+    Card,
+    CardContent,
+    Table,
+    TableBody,
+    TableCell,
+    TableHead,
+    TableRow,
+} from '@material-ui/core';
+import { Error, Loading, ShowButton, Title } from 'react-admin';
+import FilterField from '../../components/FilterField';
+import PaginationButton from '../../components/PaginationButton';
+import ListActions from '../../components/ListActions';
+import useGetList from '../../components/useGetList';
+
+const DevicesList = props => {
+    const [filter, setFilter] = useState({});
+    const [paginationCursor, setPaginationCursor] = useState(null);
+    // As the paginationCursor variable has not changed we need to force an update
+    const [seed, setSeed] = useState(Math.random());
+
+    const { data, error, loaded } = useGetList({
+        ...props,
+        filter,
+        paginationCursor,
+        seed,
+    });
+
+    const nextPage = label => {
+        setPaginationCursor(label);
+        setSeed(Math.random());
+    };
+
+    const changeFilter = (filterValue, name) => {
+        let currentFilter = filter;
+        if (filterValue) {
+            currentFilter[name] = filterValue;
+        } else {
+            delete currentFilter[name];
+        }
+        setFilter(currentFilter);
+        setPaginationCursor(null);
+        setSeed(Math.random());
+    };
+
+    if (!loaded) return <Loading />;
+    if (error) return <Error />;
+    if (!data) return null;
+
+    return (
+        <>
```

Comment:
Is this weird `<>`...`</>` construct, the fragment syntax?

Reply:
Correct, although as you have pointed out this seems have been used interchangeably with the explicit `<Fragment>` syntax. I think it may be worth being more explicit as at first glance `<>` can look like a mistake!